config/application.rb

```diff
@@ -31,9 +31,19 @@ class Application < Rails::Application
       # Setup redis as the cache_store.
       # This is required for sidekiq, which uses redis to queue jobs.
       config.cache_store = begin
-        redis_config = Rails.application.config_for(:redis).symbolize_keys
+        redis_config = Rails.application.config_for(:redis).deep_symbolize_keys
+        opts = {}
+        if redis_config[:ssl_params]
+          opts.merge!({
+                        ssl: :true,
+                        scheme: 'rediss',
```

Comment:
Is this necessary given that it's in the URL too?

Reply:
yeah, whatever way the cache_store is initialized actually takes apart the URL into the individual parts and by the time it comes to making the connection the scheme is lost unless it's explicitly passed here.